core/Visualization/Sparkline.php

```diff
@@ -181,7 +181,8 @@ private function setSparklineColors($sparkline, $seriesIndex) {
             $colors = $defaultColors;
         }
 
-        if (strtolower($colors['backgroundColor']) !== '#ffffff') {
+        if (strtolower($colors['bcore/Plugin/Controller.php
+ackgroundColor']) !== '#ffffff') {
```

Comment:
Guess that happened by accident. Nevertheless it might be useful not to simply add and commit all changes without checking them. Personally I'm always using `git add -p`, which iterates through all changes and allows to decide which one to add or not. That helps reducing such "accidents" a lot....

```suggestion
        if (strtolower($colors['backgroundColor']) !== '#ffffff') {
```